README.md

```diff
@@ -2,8 +2,8 @@
 
 |Branch|Status|
 |---|---|
-|master|[![Build status](https://ci.appveyor.com/api/projects/status/a6j46j1tawdfs3js?svg=true&branch=master)](https://ci.appveyor.com/project/appsvc/azure-webjobs-sdk-script-y8o14?branch=master)|
-|dev|[![Build status](https://ci.appveyor.com/api/projects/status/a6j46j1tawdfs3js?svg=true&branch=dev)](https://ci.appveyor.com/project/appsvc/azure-webjobs-sdk-script-y8o14?branch=dev)|
+|dev|[![Build Status](https://dev.azure.com/azfunc/Azure%20Functions/_apis/build/status/Azure.azure-functions-host?branchName=dev)](https://dev.azure.com/azfunc/Azure%20Functions/_build/latest?definitionId=37&branchName=dev)|
+|v2.x|[![Build Status](https://dev.azure.com/azfunc/Azure%20Functions/_apis/build/status/Azure.azure-functions-host?branchName=v2.x)](https://dev.azure.com/azfunc/Azure%20Functions/_build/latest?definitionId=37&branchName=v2.x)|
```

Comment:
Thanks for doing this! Can you also add release/2.0 and release/3.0, please?

Reply:
Just did. Any opinion on the ordering of the branch? I was having a hard timing deciding :)

Reply:
You already captured the order I had in mind :D